src/components/ConfigItem/ConfigItem.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from "react";
+import { ConfigItem } from "./index";
+import { ConfigDB } from "../../api/axios";
+import { MOCK_DATA } from "./storybook-mock";
+
+ConfigDB.get = (url: string) => {
+  if (url.indexOf("Subnet") > -1) {
+    return Promise.resolve({
+      data: MOCK_DATA.filter((v) => v.config_type === "Subnet")
+    });
+  } else if (url.indexOf("EC2Instance") > -1) {
+    return Promise.resolve({
+      data: MOCK_DATA.filter((v) => v.config_type === "EC2Instance")
+    });
+  } else if (url.indexOf("/config_item?id=eq.") > -1) {
+    const id = url.replace("/config_item?id=eq.", "");
+    return Promise.resolve({
+      data: [MOCK_DATA.find((item) => item.id === id)]
+    });
+  }
+};
+
+const ConfigItemDropDown = ({ type }: { type: string }) => {
+  const [selectedItem, setSelectedItem] = useState(null);
+  const [dependentSelectedItem, setDependentSelectedItem] = useState(null);
+  return (
+    <div className="flex flex-col">
+      <ConfigItem
+        className="w-96"
```

Comment:
I think this should be defaulted - we may also need to add support for labels, and descriptions for the fields. otherwise it might be hard to fit this into a form. 

The dependants should also still render when the parent is null, but in a disabled state - it makes the ui more discoverable, and less prone to dialog resizing etc.

Reply:
I am working on moving this to child way of loading that will solve and provide support for all the use cases mentioned above.